Wikipedia/UI-V5/WMFHomeViewController.m

```diff
@@ -79,13 +78,21 @@ - (NSString*)title {
 
 #pragma mark - Accessors
 
+- (void)setSavedPages:(MWKSavedPageList* __nonnull)savedPages {
```

Comment:
This line should be unnecessary because this object requires that properties are set before they are used See line 247:

```
 NSParameterAssert(self.dataStore);
 NSParameterAssert(self.searchSite);
 NSParameterAssert(self.recentPages);
 NSParameterAssert(self.savedPages);
```

Nothing should be created (lazy loading) until viewDidAppear. `configureDatasource` isn't called until then.